### PR TITLE
OEP7: URI schema and sample code

### PR DESCRIPTION
**Pull request by JasonZhouPW**
add OEP-7 propsal and sample python code

### CODE REVIEW

File: OEP-7/OEP-7 URI Schema.md [hunk 1]
```diff
@@ -0,0 +1,233 @@
...
+def registerDomain(fulldomain, registerdid,idx, validto)
+```
+
+This allows user to register a domain ,like standard URL, domain format can be *.a.ont, only contract admin can register the top level domain, top domain owner can register the sub level domain and this should trigger a "registerDomain" event.
+
+registerdid is user's ONTID and the idx is user's wallet index of the ONTID.
+
+the  fulldomain should be alphanumeric and no more than 253 characters, and every part should no more than 63 characters.
+
+valid_to is time stamp
```
Comment (tanZiWen) [inline OEP-7/OEP-7 URI Schema.md:64]: expire time?

Comment (tanZiWen) [reply OEP-7/OEP-7 URI Schema.md:64]: explain idx ?

Comment (tanZiWen) [reply OEP-7/OEP-7 URI Schema.md:64]: whether every parameters have type distinguish?

**Comment by JasonZhouPW** [comment]
main net contract address: a79d8dcab5a3a3b0b38787205c0f8f8b67ee68c4

**Pull request merged**